pom.xml

```diff
@@ -30,7 +30,8 @@
     <maven.compiler.target>1.6</maven.compiler.target>
     <commons-validator.version>1.6</commons-validator.version>
     <inspektr.version>1.2.GA</inspektr.version>
-    <ldaptive.version>1.0.6</ldaptive.version>
+    <ldaptive.version>1.2.1</ldaptive.version>
+    <spring.version>4.2.0.RELEASE</spring.version>
```

Comment:
we only use Spring for tests. the version we had been using (3.2.1.RELEASE) does not support Java 8. https://jira.spring.io/browse/SPR-10292